core-feature-pack/galleon-common/src/main/resources/layers/standalone/remoting/layer-spec.xml

```diff
@@ -5,10 +5,4 @@
         <layer name="elytron"/>
     </dependencies>
     <feature-group name="remoting"/>
-
-    <feature spec="subsystem.remoting.http-connector">
-        <param name="http-connector" value="http-remoting-connector"/>
-        <param name="connector-ref" value="default"/>
-        <param name="sasl-authentication-factory" value="application-sasl-authentication"/>
-    </feature>
```

Comment:
Note to self:

This change requires https://github.com/wildfly/wildfly/pull/17145/files as that is where the remoting feature group used in full WF that provides the http-connector is configured. The 'remoting' feature group file here is not relevant in full WF.

Reply:
Correct.